frontend/src/pages/history/components/cardlist/cardlist.tsx

```diff
@@ -0,0 +1,146 @@
+import styled from '@emotion/styled'
+import {useState} from 'react'
+
+import {Card} from '@/components/card'
+import {Tab} from '@/components/tab'
+import {useFetch} from '@/hooks/use-fetch'
+import {API_BASE_URL} from '@/utils/constants'
+
+export interface StudentReservation {
+  instructorId: number
+  reservationId: number
+  instructorImage: string
+  instructorName: string
+  academyName: string
+  reservationDate: string
+  reservationTime: number
+  trainingTime: number
+}
+
+export type StudentHistoryResponse = StudentReservation[]
+
+const Container = styled.div({
+  display: 'flex',
+  flexDirection: 'column',
+  flexGrow: 1,
+  overflowY: 'hidden',
+})
+const List = styled.div({
+  display: 'flex',
+  overflowY: 'auto',
+  flexDirection: 'column',
+  padding: '2rem',
+  gap: '1rem',
+})
+const Label = styled.p(({theme}) => ({
+  fontWeight: 700,
+  fontSize: '2rem',
+  color: theme.color.gray800,
+  padding: '2rem',
+}))
+
+interface StudentCardlistProps {
+  onReviewClick: (item: StudentReservation) => void
+  selected?: StudentReservation
+}
+
+function useStudentList() {
+  // todo: infinite scroll
+  return useFetch<StudentReservation[]>(`${API_BASE_URL}/my/reservations?pageNumber=1&pageSize=10`)
+}
+
+export function StudentCardlist({onReviewClick, selected}: StudentCardlistProps) {
+  const {data} = useStudentList()
```

Comment:
data 같은 이름보다는 명시적으로 이름을 지어주어도 괜찮을거 같아요

```suggestion
  const {data: studentList} = useStudentList()
  
  // use case
  studentList.map(...)
```

아니면 이렇게도 해볼 수 있겠네요. 아래처럼 하면 객체로 묶는 효과가 있어서 코드를 읽을 때 어떤 API의 data, loading인지 알기 쉬워요
```suggestion
  const studentList = useStudentList()
  
  // 사용할 때는
  studentList.data.map(...)
  studentList.loading
```